configure.sh

```diff
@@ -124,7 +124,7 @@ build:nohdfs --define=no_hdfs_support=true
 build:nonccl --define=no_nccl_support=true
 
 build --config=v2 --config=noaws --config=nogcp --config=nohdfs --config=nonccl
-test --config=v2 --compilation_mode dbg
+test --config=v2 --compilation_mode=fastbuild
```

Comment:
I tried to enable this in the past for the MLIR converter tests, but it failed or segfaulted if I remember correctly. However we change quite a bit in the testing setup since then, so if CI is happy this looks great.

Reply:
Yeah it seems to work fine, although doesn't look like it makes the MLIR builds much (if at all) quicker. Definitely helps the other test builds though.